networkx/classes/function.py

```diff
@@ -288,33 +292,51 @@ def info(G, n=None):
 
 
 def set_node_attributes(G, name, values):
-    """Set node attributes from dictionary of nodes and values
+    """Sets node attributes from a given value or dictionary of values.
 
     Parameters
     ----------
     G : NetworkX Graph
 
     name : string
-       Attribute name
+       Name of the node attribute to set.
 
-    values: dict
-       Dictionary of attribute values keyed by node. If `values` is not a
-       dictionary, then it is treated as a single attribute value that is then
-       applied to every node in `G`.
+    values : dict
+       Dictionary of attribute values keyed by node. If ``values`` is
```

Comment:
I have a question about formatting docstrings. I  thought we were supposed to use single quotes around parameter values and save double quotes for literal code.  I finally tracked it down and its the [numpydoc page](https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt#common-rest-concepts) that I got it from.

Is there a reason to go with ``` ``values`` ```?  I feel it interrupts the text flow (also I don't like how it makes the text red in the sphinx docs, but that's a different issue. :)   I'm still learning (always learning?) the formatting traditions/standards.